src/sentry/web/frontend/accounts.py

```diff
@@ -97,6 +97,7 @@ def recover_confirm(request, user_id, hash):
             form = ChangePasswordRecoverForm(request.POST)
             if form.is_valid():
```

Comment:
Nah, this is good catch. Worst thing is that the user would have just been redirected back to login. And probably literally never would have noticed. I've bound it to the request explicitly.